code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/CreateEventFragment.java

```diff
@@ -80,27 +89,28 @@ public int getCount() {
             String category = categoryInput.getSelectedItem().toString();
             String location = locationInput.getText().toString().trim();
             Boolean tracking = geoTracking.isChecked();
+            //Timestamp timestamp = ((AttendeeMainActivity) requireActivity()).getTimestamp();
```

Comment:
Can we get rid of this completely instead of commenting it out?

Reply:
yeah, sorry about that